packages/plugin-core/src/commands/base.ts

```diff
@@ -64,7 +65,7 @@ export abstract class BaseCommand<
 
   abstract enrichInputs(inputs: TGatherOutput): Promise<TOpts | undefined>;
 
-  abstract execute(opts: TOpts): Promise<TOut>;
+  abstract execute(opts?: TOpts): Promise<TOut>;
```

Comment:
hmm, was this a stray change?

Reply:
This one was purposeful. 

We have commands that do not require options when they execute. Example `GoToNote`, So now with CommandFactory we are calling execute through the interface of BaseCommand. 

(As example we want to allow ShowPreview to call GoToNote command but we dont want ShowPreview to know the exact implementation of GoToNote).